io.github.opennumismat.open-numismat.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.opennumismat.open-numismat
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: '6.8'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: open-numismat
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
+  - --filesystem=/media      # for export to mobile
+  - --filesystem=/run/media  # for export to mobile

```

Comment:
what is "for export to mobile" ?

Reply:
User can export part of collection to different format for mobile version